mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -450,6 +551,23 @@ properties:
                           name: 'radix'
                           description: |
                             The native way to select the alphabet. Must be in the range \[2, 95\].
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'replaceDictionaryConfig'
```

Comment:
Can you please add a test example for this field? Thanks.

Reply:
Actually, I've already added create-update test of this field in an existing test function namely  `TestAccDataLossPreventionDeidentifyTemplate_dlpDeidentifyTemplate_infoTypeTransformationsUpdate`. The config where it is updated can be found [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/7816/files#diff-ec31eff9a8a9a675959b0a56dbf6b1b45df815865408a2cad9a9bb4c6026e451R240) for create and [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/7816/files#diff-ec31eff9a8a9a675959b0a56dbf6b1b45df815865408a2cad9a9bb4c6026e451R409) for update.